streams/cn.m3u

```diff
@@ -1025,27 +1025,27 @@ http://ott.js.chinamobile.com/PLTV/3/224/3221227393/index.m3u8
 https://hw-m-l.cztv.com/channels/lantian/channel01/1080p.m3u8
 #EXTINF:-1 tvg-id="ZhejiangSatelliteTV.cn",浙江卫视 (576p)
 http://183.207.248.71/gitv/live1/G_ZHEJIANG/G_ZHEJIANG
-#EXTINF:-1 tvg-id="",浙江国际
+#EXTINF:-1 tvg-id="ZheJiang-Int-2.cn",浙江国际
 https://ali-m-l.cztv.com/channels/lantian/channel010/1080p.m3u8
-#EXTINF:-1 tvg-id="",浙江国际 (1080p)
+#EXTINF:-1 tvg-id="ZheJiang-Int3.cn",浙江国际 (1080p)
 https://hw-m-l.cztv.com/channels/lantian/channel010/1080p.m3u8
-#EXTINF:-1 tvg-id="",浙江少儿
+#EXTINF:-1 tvg-id="ZheJiang-Kids2.cn",浙江少儿
 https://ali-m-l.cztv.com/channels/lantian/channel008/1080p.m3u8
 #EXTINF:-1 tvg-id="ZhejiangChildrensChannel.cn",浙江少儿 (1080p)
 https://hw-m-l.cztv.com/channels/lantian/channel008/1080p.m3u8
```

Comment:
ZhejiangEducationTechnologyChannel.cn per the image above.

Alt https://qiniup-m-l.cztv.com/channels/lantian/channel008/1080p.m3u8
or https://ct-m-l.cztv.com/channels/lantian/channel008/1080p.m3u8